packages/angular/src/schematics/application/application.spec.ts

```diff
@@ -353,51 +353,58 @@ describe('app', () => {
         );
 
         const eslintConfig = readJsonInTree(tree, 'apps/my-app/.eslintrc.json');
-
-        expect(eslintConfig.overrides).toMatchInlineSnapshot(`
-          Array [
-            Object {
-              "extends": Array [
-                "plugin:@nrwl/nx/angular",
-                "plugin:@angular-eslint/template/process-inline-templates",
-              ],
-              "files": Array [
-                "*.ts",
-              ],
-              "parserOptions": Object {
-                "project": Array [
-                  "apps/my-app/tsconfig.*?.json",
+        expect(eslintConfig).toMatchInlineSnapshot(`
```

Comment:
This diff is purely the change to make the snapshot to capture the whole config, instead of just `overrides`